### DISCUSSION TITLE
Allow custom redirects for Job completions

### DISCUSSION THREAD

**Discussion by jmcgill298** [nautobot/nautobot#628] [Ideas]
It would be nice to be able to provide a custom redirect page for successfully completed jobs. Using the first example from the Jobs Documentation: `Automatically populate new devices and cables in preparation for a new site deployment`

I would like to provide a pretty view of all created objects and formed relationships, and then also provide an export button that users can easily distribute the information to relevant teams (such as Data Center team to rack and cable).

I think that several use cases for Jobs could benefit from having a custom redirect, instead of the current Job Result View.

**Comment by glennmatthews** [comment]
I agree that this would be useful. I have a similar requirement in a plugin I'm working on, of redirecting to a view that provides more job-specific information than the generic JobResult view can provide. For the time being I'm working around this by having my plugin provide a CustomLink on the JobResult view that links to my detailed view, but it would be even better from a UI/UX perspective if this could be an automatic redirect.
